src/test/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseControllerIntTest.java

```diff
@@ -656,4 +666,110 @@ void GET_cases_givenNoCreatedFilterParams_whenGetCases_thenReturnAllCases() {
+                .oauth2(getToken())
+                .when()
+                .get("/defendant/{defendantId}/offender", DEFENDANT_ID)
+                .then()
+                .assertThat()
+                .statusCode(200)
+                .body("crn", equalTo("X320741"))
+                .body("probationStatus", equalTo("CURRENT"))
+                .body("previouslyKnownTerminationDate", equalTo("2010-01-01"))
+                .body("awaitingPsr", equalTo(true))
+                .body("breach", equalTo(true))
+                .body("preSentenceActivity", equalTo(true))
+                .body("suspendedSentenceOrder", equalTo(true))
+            ;
+        }
+
+        @Test
+        void DELETE_offender_givenDefendantId_whenDeleteOffender_thenRemoveAssociation() {
+
+            String DEFENDANT_ID_FOR_DELETE = "9c2f11b0-1bca-4b24-85a1-315d67020b2c";
+            given()
+                .auth()
+                .oauth2(getToken())
+                .when()
+                .delete("/defendant/{defendantId}/offender", DEFENDANT_ID_FOR_DELETE)
+                .then()
+                .assertThat()
+                .statusCode(200);
+            final var defendant = defendantRepository.findFirstByDefendantIdOrderByIdDesc(DEFENDANT_ID_FOR_DELETE).get();
+            assertThat(defendant.getCrn()).isNullOrEmpty();
+        }
+
+        @Test
+        void PUT_offender_givenDefendantIdAndUpdatedOffender_whenPutOffender_thenUpdateOffender() {
+
+            final String offenderUpdate = "{\n" +
+                    "                \"crn\": \"Y320741\",\n" +
+                    "                \"probationStatus\": \"NOT_SENTENCED\",\n" +
+                    "                \"previouslyKnownTerminationDate\": \"2010-01-01\",\n" +
+                    "                \"awaitingPsr\": true,\n" +
+                    "                \"breach\": false,\n" +
+                    "                \"preSentenceActivity\": true,\n" +
+                    "                \"suspendedSentenceOrder\": true\n" +
+                    "            }";
+
+            String DEFENDANT_ID_FOR_UPDATE = "d59762b6-2da7-4af0-a09f-7296d40f15ce";
+            given()
+                .auth()
+                .oauth2(getToken())
+                .contentType(ContentType.JSON)
+                .accept(ContentType.JSON)
+                .when()
+                .body(offenderUpdate)
+                .put("/defendant/{defendantId}/offender", DEFENDANT_ID_FOR_UPDATE)
+                .then()
+                .assertThat()
+                .statusCode(200);
+
+            final var offender = offenderRepository.findByCrn("Y320741").get();
+            assertThat(offender.getProbationStatus()).isEqualTo(OffenderProbationStatus.NOT_SENTENCED);
+            assertThat(offender.isBreach()).isFalse();
+        }
+
+        @Test
+        void PUT_offender_givenDefendantIdWithNoOffenderAndGivenNewOffender_whenPutOffender_theCreateAndAssocaiteOffender() {
```

Comment:
I'm amazed I even spotted this typo 😆 

```suggestion
        void PUT_offender_givenDefendantIdWithNoOffenderAndGivenNewOffender_whenPutOffender_theCreateAndAssociateOffender() {
```